### DISCUSSION TITLE
Allow customization of the footer signature.

### DISCUSSION THREAD

**Discussion by dvarrazzo** [pradyunsg/furo#315] [Ideas] [answered]
I don't want to be insensitive, but I think that the generated page footer focus about Furo a bit too much.

![image](https://user-images.githubusercontent.com/199429/143146214-f0485317-1496-4b54-91ca-99f9d3b5a46d.png)

I have chosen not to spam the footer with links to my personal blog, and I would like to have the opportunity to choose what to display about Furo, without overriding the whole `page.html`.

If I had the choice I would still link to the Furo project page, but also to Spninx, and would like to give the footer my wording.

Thank you for understanding :slightly_smiling_face:

**Comment by pradyunsg** [comment]
You'll want to unset `html_show_sphinx` in your conf.py, if you want a callout to Sphinx.

As for the footer being difficult to override, that is intentional. I've easily spent a few hundred hours on this project, and I think it's reasonable to have a callout on pages that use it -- not least because it shows to users that the underlying theme is reusable and can be found.

**Reply by dvarrazzo** [reply]
I know, and I would respect that by keeping the link to the Furo project in the footer: I have no intention to remove it. But, for my taste, which is entirely personal, the link to your personal blog and your twitter handle is a bit too much: I have spent thousands of hours on my project and you can see that there is no link to my blog (I have one) or to my Github. This is _a choice_.

Even the top banner you can see in our docs (https://www.psycopg.org/psycopg3/docs/), the call for sponsor, is not part of the doc: it is an overlay added when the website is generated. Documentation is a serous matter for us, more serious than advertisement, and I don't wish to have content in the docs which is not _exactly_ what I wish to have.

**Reply by pradyunsg** [reply]
Ah, I completely misunderstood what you were asking for! Whoops!

**Comment by pradyunsg** [comment]
Yea, so... The footer of this theme is probably the main element that I wish I had a better idea for what to do with. 😅 

There's basically three things in there right now:

- the copyright: there was no other place to put it, realistically.
- the callout for Sphinx and this theme: see [other comment](https://github.com/pradyunsg/furo/discussions/315#discussioncomment-1692267)
- a "show source" button: this does totally the wrong thing, from what I can understand after talking to most users. :) 

And I have three questions for you:

- What sort of control would you like to have on the footer's contents?
- What content do you want to add or exclude from the footer?
- Do you have any suggestions on what you'd like the footer to look like?

Specificity helps here, and a screenshot + editted HTML would be ideal!

**Reply by dvarrazzo** [reply]
I have something like this in mind:

![image](https://user-images.githubusercontent.com/199429/143788840-8a30ad15-de65-4ffd-9815-ee220eeb0f66.png)

I dropped the link to the page source, and added the link to Sphinx, using the relative conf.py settings.

**Reply by francois-rozet** [reply]
Hello @pradyunsg, love your work but as @dvarrazzo said, I find the link to your personal website a bit much when I don't even include mine in my own projects. Letting the `Furo` link is fine though. For instance,

<img width="714" alt="image" src="https://user-images.githubusercontent.com/37352336/153418968-ac356573-1f16-4a43-939c-6312c2d58f2a.png">

By the way, I do agree that the "show source" button is misleading. Most user don't care about the `rst` files but want the package code.

**Comment by pradyunsg** [comment, marked as answer]
See #374, for providing feedback on a footer redesign!
